components/admin_console/request_button/request_button.jsx

```diff
@@ -126,6 +131,7 @@ export default class RequestButton extends React.Component {
     }
 
     static defaultProps = {
+        id: null,
```

Comment:
Not sure if this is the "proper" way to go about dynamically adding an id if one is provided or not.

Reply:
We don't need to pass a default value for something like null unless were expecting id to be that as default instead of undefined.

Reply:
Oh right... I did focus on the `id` (hence my reply) and not the value of `null`.